worlds/hk/__init__.py

```diff
@@ -218,11 +247,58 @@ def create_items(self):
             for loc, price in zip(locations, prices):
                 loc.cost = price
 
+    def _can_beat_thk(self, state, player):
```

Comment:
Does this belong here? Shouldn't this be a part of `HKLogicMixin`?

Reply:
This should now be done.  HKLogicMixin didn't exist when I first opened this, hopefully I used it correctly.